frontend/src/metabase/parameters/utils/mapping-options.unit.spec.js

```diff
@@ -9,11 +9,15 @@ import {
 import { getParameterMappingOptions } from "./mapping-options";
 
 function structured(query) {
-  return SAMPLE_DATABASE.question(query).card();
+  const card = SAMPLE_DATABASE.question(query).card();
+  card.id = 1;
```

Comment:
Not sure what setting the id does but reverting this change still pass the tests for me.

Reply:
Hm, I think a prior version of this PR broke these tests, but now they no longer break. Likely related to this commit https://github.com/metabase/metabase/pull/24645/commits/e2ec3c6621b32b9a0a751a654ab554b41fd248fe. I've reverted the change.